examples/vector_fe/vector_fe_ex3/run.sh

```diff
@@ -6,17 +6,29 @@ source $LIBMESH_DIR/examples/run_common.sh
 
 example_name=vector_fe_ex3
 
-# Note: these problems are particularly ill-conditioned, and common
-# preconditioners like ILU don't all work very well.  We currently
-# force Jacobi in these examples.
+# Note: these problems are particularly ill-conditioned, so we currently
+# resort to a direct solver.
 
-options="element_type=TRI6 -pc_type jacobi"
+options="element_type=TRI6 -pc_type lu"
```

Comment:
It'll use MUMPS, at least on my machine, might depend on what you configure with, not sure.
I don't know since when, but it does work nicely in parallel.